kentik_api_library/kentik_api/analytics/data_frame_cache.py

```diff
@@ -148,7 +148,10 @@ def info(self, out=None):
         print(f"newest data: {self.newest}", file=out)
 
     def files_in_range(
-        self, start: Optional[datetime] = None, end: Optional[datetime] = None, contained: bool = False
```

Comment:
I guess most of this changes is because of pylint's default max line length of 100, which is unnecessary. I'd like to be wrong, tough.

Reply:
The reason of the changes is black linter.
Here You have a log about it: https://github.com/kentik/community_sdk_python/runs/6571203458?check_suite_focus=true#step:6:7255

Reply:
As discussed, let's ignore this comment